docs/data/base/components/menu/menu.md

```diff
@@ -9,78 +9,55 @@ waiAria: https://www.w3.org/WAI/ARIA/apg/patterns/menu-button/
-      <Menu>
-        <MenuItem>My account</MenuItem>
-        <MenuItem>Notification preferences</MenuItem>
-      </Menu>
-    </Dropdown>
-  );
-}
 ```
 
-### Basics
-
-The Menu replaces the native HTML `<ul>`, and the Menu Item corresponds to the `<li>` tag.
-
-The visibility of the menu is controlled by the Menu Button.
-
-The following demo shows how to create and style a Menu component.
+The demo below shows how to create and style a Dropdown Menu.
 Click **Dashboard** to view the menu.
-Notice that it uses the built-in [Popper](/base-ui/react-popper/) component to break out of its parent container visually:
+Note that it uses the built-in [Popper](/base-ui/react-popper/) component to visually break out of its parent container:
 
 {{"demo": "MenuSimple"}}
 
-The Dropdown should be the outermost component—all other menu-related components must be placed as its children (not necessarily as direct ones).
-If you need to control the open state of the menu or react to its changes, place an `open`/`onOpenChange` props on the Dropdown.
+The `<Dropdown />` should be the outermost component—all other Menu-related components must be placed as its children (but not necessarily as direct ones).
+If you need to control the open state of the Menu or react to its changes, place an `open`/`onOpenChange` props on the `<Dropdown />`.
 
-The Dropdown should contain at most one Menu Button and one Menu.
-It will wire them together, so that pressing the button will open the menu.
-It also takes care of assigning proper accessibility attributes, so the menu can be used with assistive technologies or a keyboard.
+The `<Dropdown />` must only contain one `<MenuButton />` and one `<Menu />`.
+It will wire them together, so that pressing the Button will open the Menu.
+It also takes care of assigning proper accessibility attributes, so the Dropdown Menu can be used with assistive technologies or a keyboard.
 
-The Menu hosts Menu Items.
-They can be wrapped in arbitrary tags and components or even grouped together.
-Clicking on a menu item closes the associated menu.
+The `<Menu />` hosts `<MenuItem>` components.
+They can be wrapped in arbitrary tags and components, as well as grouped together.
+Clicking on a Menu Item closes its associated Menu.
 
 ### Anatomy
 
-- The Menu component comprises a root slot that renders an `<div>` by default and a listbox slot that is a `<li>`.
-- The Menu Item has a single root `<li>` slot.
-- The Menu Button is a native HTML `<button>`.
-- The Dropdown does not render any HTML element.
-  It only provides a context that links Menu Button and Menu.
-  Thanks to this context, developers do not have to wire these components using custom logic.
+- The Dropdown does not render any HTML element—it only provides the context that links a Menu Button to a Menu, so you don't have to.
+- The Menu Button renders a `<button>`.
+- The Menu component renders a `<div>` with a `<ul>` nested inside.
+- The Menu Item renders a `<li>`.
```

Comment:
I figured it made sense to use `<ComponentCase />` here anytime it might be unclear whether we're talking about a specific component rather than the concept _or_ the "suite" of Menu components as a whole. I didn't want to use it everywhere, unless that helps clarify things. Now that I think about it more, we can't assume that a reader has already read other sections of the page, so we should err on the side of using it more. 👍